public/app/plugins/datasource/tempo/VariableQueryEditor.test.tsx

```diff
@@ -55,7 +55,14 @@ describe('TempoVariableQueryEditor', () => {
     expect(onChange).not.toHaveBeenCalled();
     render(<TempoVariableQueryEditor {...props} onChange={onChange} />);
 
+    // Select the query type ('Label values')
```

Comment:
considering the commented line has words`selectOption` + `'Query type'` + `'Label values'` I don't think this comment adds anything.